kmcd-driver.md

```diff
@@ -198,9 +198,12 @@ We simulate that here.
 
 ```k
     syntax MCDStep ::= "TimeStep"
- // -----------------------------
-    rule <k> TimeStep => . ... </k>
-         <current-time> TIME => TIME +Int 1 second </current-time>
+                     | "TimeStep" Int
+ // ---------------------------------
+    rule <k> TimeStep => TimeStep 1 ... </k>
+
+    rule <k> TimeStep N => . ... </k>
+         <current-time> TIME => TIME +Int N </current-time>
```

Comment:
It might be a good idea to add the condition that N is positive to eliminate the possibility of having behaviors where time is reversed or behaviors with vacuous time steps (this could be especially important for random testing).

Reply:
Done.